docs/FAQ.md

```diff
@@ -0,0 +1,23 @@
+# FAQ for s2n-tls
+
+### Why isn't my connection using TLS1.3?
+There are several reasons why the TLS protocol version may not be what you expected. One is that the TLS protocol has some legacy version fields that have ossified, making it more difficult to understand what protocol was negotiated. The TLS1.3 protocol sets the ClientHello version to TLS1.2 for backwards compatibility with poorly implemented TLS servers and middleboxes. The record version is set to TLS 1.1 in the ClientHello for the same reason. There is a separate ClientHello extension called "supported_versions" which lists the actual versions supported, included TLS1.3. See the [protocol version section](USAGE-GUIDE/#protocol-version) to check the real protocol negotiated.
+Additionally you need a security policy that supports TLS1.3 in order to negotiate TLS1.3. See [security policies](USAGE-GUIDE.md/#security-policies).
```

Comment:
I wonder if we can try to format this to make the different possibilities clearer:
```suggestion
### Why isn't my connection using TLS1.3?
There are several possible reasons:
* Are you using a security policy that supports TLS1.3? See [security policies](USAGE-GUIDE.md/#security-policies).
* Are you verifying the connection version correctly? If you're examining the version in a packet capture, make sure that you're checking the right version field. The TLS protocol includes several legacy version fields that have ossified, making it more difficult to understand what version was negotiated. TLS1.3 sets the record version field to TLS1.1 or TLS1.2 and the ClientHello version field to TLS1.2 for backwards compatibility reasons. There is a separate ClientHello extension called "supported_versions" which lists the actual versions supported, included TLS1.3. See the [protocol version section](USAGE-GUIDE/#protocol-version) for methods to check the actual protocol version negotiated.
* Are you using a libcrypto library that supports TLS1.3? Modern libcrypto libraries support the algorithms needed for TLS1.3, but older libraries like Openssl 1.0.2 do not. If s2n-tls is built with Openssl 1.0.2, TLS1.3 is unlikely to be negotiated. 
* Does your peer support TLS1.3? If your peer does not support TLS1.3, TLS1.3 will not be negotiated.
```

Reply:
Thanks, this is a good edit.